tests/jballerina-unit-test/src/test/resources/test-src/services/client_resource_call_negative.bal

```diff
@@ -0,0 +1,62 @@
+type CustomString string;
+type CustomString2 CustomString;
+
+string? stringNilArgument = ();
+() nilArgument = ();
+
+CustomString customString = "x";
+CustomString2 customString2 = customString;
+
+const int intConstant = 1;
+var varInt = 1;
+var varBoolean = true;
+
+public function testClientResourceFunctionCallError() {
+    var successClient = client object {
+        resource function get path() returns int {
+            return 1;
+        }
+
+        resource function post path() returns int {
+            return 1;
+        }
+
+        resource function get path/[int a]() returns int {
+            return 1;
+        }
+
+        resource function get path/[int a]/foo(string name) returns int {
+            return 1;
+        }
+
+        resource function get path/[int a]/foo2(string name, string address) returns int {
+            return 1;
+        }
+    };
+
+    int result = successClient->/path/[10*11]/foo(123);
+    result = successClient->/path/[1]/foo("x", "y"); // No Error occurred
+    result = successClient->/path/[1]/foo("x", "y", "z");
```

Comment:
@rdulmina, we are getting `too many arguments in call to 'get()'`
But there are few get methods. Can't we specify what is the exact method?
Like,
`too many arguments in call to 'get path/[int a]/foo2(string name, string address)'`

Reply:
Will have a look

Reply:
Created issue to address this later https://github.com/ballerina-platform/ballerina-lang/issues/36709